create_jenkins_job.py

```diff
@@ -218,9 +211,10 @@ def create_job(os_name, job_name, template_file, additional_dict):
             'cmake_build_type': 'Release',
             'time_trigger_spec': PERIODIC_JOB_SPEC,
             'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
-            'use_connext_default': 'false',
-            'use_fastrtps_default': 'true',
-            'use_opensplice_default': 'true',
+            'ignore_rmw_default': {
```

Comment:
That would imply that all RMW implementations are known. Without requiring that information it is possible to feed a `.repos` with additional RMW implementations.

Reply:
I mean within the scope of the `create_jenkins_job.py` script, for the purpose of populating the `ignore_rmw_default` value.

Reply:
> Since the purpose of this job has expanded now, its job name should be updated

Can you clarify what you mean with this. The job is still building OpenSplice and tests against one other RMW impl - also still FastRTPS. So I am not sure what has changed.

Reply:
> I mean within the scope of the `create_jenkins_job.py` script, for the purpose of populating the `ignore_rmw_default` value.

Maybe I can't see how you imagine it to look like. Can you either describe it more concretely or propose a patch to clarify.

Reply:
> Can you clarify what you mean with this. The job is still building OpenSplice and tests against one other RMW impl - also still FastRTPS. So I am not sure what has changed.

Yep: currently the job has ospl in the name because that's its purpose: it's the only nightly with opensplice. Now it has an additional purpose: it's the only nightly with fastrtps dynamic, so the name should reflect that

Reply:
I see why we might be on different pages. The code as it stands is ignoring fastrtps_dynamic. My understanding (and the summary that is in the PR description) is that fastrtps dynamic should be _enabled_ in a nightly with opensplice for each platform, including tests. In my head that is this job that we are discussing (perhaps that's where we disagree). Otherwise, fastrtps dynamic is only enabled in packaging jobs and the xenial job, right?

This gets at my motivation for wanting to better capture our intent (otherwise the reader has to do the arithmetic themselves). It would look something like this:
```
known_rmws = {'rmw_fastrtps_cpp', 'rmw_fastrtps_dynamic_cpp', 'rmw_connext_cpp', 'rmw_connext_dynamic_cpp', 'rmw_opensplice_cpp'}
enabled_rmws = {'rmw_opensplice_cpp', 'rmw_fastrtps_dynamic_cpp'}
ignore_rmw_default = known_rmws - enabled_rmws
```